testsuite/integration-arquillian/tests/other/console/src/test/java/org/keycloak/testsuite/console/authentication/FlowsTest.java

```diff
@@ -214,20 +205,24 @@ public void navigationTest() {
         assertAlertSuccess();
 
         List<String> expectedOrder = new ArrayList<>();
-        Collections.addAll(expectedOrder, "Kerberos", "Cookie", "Copy Of Browser Forms", "OTP Form",
-                                          "Username Password Form", "Identity Provider Redirector");
-
+        Collections.addAll(expectedOrder,
```

Comment:
You can use a single instruction: `List<String> expectedOrder = Arrays.asList(value1, value2, ..., valueN);`